Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2207,78 +2092,79 @@ - (UIWritingToolsAllowedInputOptions)writingToolsAllowedInputOptions {
 }
 #endif
 
-- (BOOL)_wantsCompleteUnifiedTextReplacementBehavior
-{
-    return [self wantsWritingToolsInlineEditing];
-}
-
 - (BOOL)wantsWritingToolsInlineEditing
 {
-    return [self _isEditable] || [_configuration _unifiedTextReplacementBehavior] == _WKUnifiedTextReplacementBehaviorComplete;
+#if PLATFORM(IOS)
+    return [self _isEditable] || [_configuration writingToolsBehavior] == UIWritingToolsBehaviorComplete;
+#elif PLATFORM(MAC)
+    return [self _isEditable] || [_configuration writingToolsBehavior] == NSWritingToolsBehaviorComplete;
```

Comment:
Can we also use your utilities file to unify `UIWritingToolsBehaviorComplete` / `NSWritingToolsBehaviorComplete`?